dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-zookeeper/src/main/java/org/apache/dolphinscheduler/plugin/registry/zookeeper/ZookeeperAuthSchema.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.dolphinscheduler.plugin.registry.zookeeper;
+
+public enum ZookeeperAuthSchema {
+    DIGEST,
+}
```

Comment:
Missing license, and can we removed this enum? We don't need to maintain a enum here.